flink-autoscaler/src/main/java/org/apache/flink/autoscaler/JobVertexScaler.java

```diff
@@ -245,50 +247,69 @@ private boolean detectIneffectiveScaleUp(
         }
     }
 
+    /**
+     * Compute newParallelism according to currentParallelism.
+     *
+     * @param currentParallelism The current parallelism.
+     * @param maxParallelism The max parallelism for job vertices. It's numKeyGroups by default, and
+     *     it's partition number for kafka source vertex.
+     * @param scaleFactor The scale factor.
+     * @param parallelismLowerLimit The parallelism lower limitation in autoscaler option.
+     * @param parallelismUpperLimit The parallelism upper limitation in autoscaler option.
+     * @param adjustByMaxParallelism True means we need to adjust parallelism according to the
+     *     maxParallelism to ensure the keyGroup or partition evenly.
+     */
```

Comment:
I prefer adding just an explanation of what this method does. There is no additional information which is not already in the method signature. If we want to clarify, e.g. maxParallelism, we can add a dedicated comment for that.

Reply:
(I'm suggesting to remove the `@param` lines)

Reply:
Awesome, thanks!